src/App.tsx

```diff
@@ -30,7 +30,7 @@ function App() {
       OpenAPI.TOKEN = import.meta.env.VITE_TOKEN;
     }
     whoami();
-    executeTest();
```

Comment:
?

Reply:
The app will not render the result on the page when the app first enters.

---

The fun `executeTest ` is an async function.
We must keep waiting for the result in the `useEffect`.

More detail about.
https://devtrium.com/posts/async-functions-useeffect#write-the-asynchronous-function-inside-the-useeffect